### ISSUE TITLE
kvasir dataset results in the paper

### ISSUE THREAD

**Issue by uobinxiao** [tianrun-chen/SAM-Adapter-PyTorch#38]
hello, as mentioned in the paper, this dataset follow the evaluation protocol of mediaeval 2020: Automatic polyp segmentation. However, their test set doesn't contain ground truth. And the dataset we can download from their website contains a training set and val set. Are the results reported in the paper are generated by the val set?
